### ISSUE TITLE
OVH provider fail to update with "nohost" answer

### ISSUE THREAD

**Issue by jaytonic** [qdm12/ddns-updater#639]
**TLDR**: *Describe your issue in a one liner here*
I've had a ddns updater configured with ovh(dynhost) method for the last 6 month, it just stopped working with a "Nohost" answer

1. Is this urgent: Yes
2. DNS provider(s) you use: ovh
3. Program version: 2.6.0
4. What are you using to run the container: docker-compose
5. Extra information (optional)

Logs:

```log
2024-02-07T19:22:26Z DEBUG configured to fetch IP: v4 or v6: false, v4: true, v6: false
2024-02-07T19:22:26Z DEBUG your public IP address are: v4 or v6: invalid IP, v4: xx.xx.47.241, v6: invalid IP
2024-02-07T19:22:26Z INFO ipv4 address of sub.domain.com.domain.com is xx.xx.32.123 and your ipv4 address  is xx.xx.47.241
2024-02-07T19:22:26Z INFO Updating record [domain: domain.com | host: sub.domain.com | provider: ovh | ip: ipv4] to use xx.xx.47.241
2024-02-07T19:22:26Z DEBUG GET https://username:password@0www.ovh.com/nic/update?hostname=subdomain.domain.com.domain.com&system=dyndns | headers: User-Agent: DDNS-Updater quentin.mcgaw@gmail.com; Authorization: Basic XXX==
     
2024-02-07T19:22:27Z DEBUG 200 OK | headers: X-Iplb-Request-Id: 0A6EC06E:14C6_8D5FBAC8:01BB_65C3D872_DB300990:36217,894A789E:B3D2_C063418B:01BB_65C3D872_C5D5C:7D8B; X-Cacheable: Not cacheable: no cache; X-Cdn-Pop: rbx; X-Request-Id: 848233193; Cache-Control: max-age=60; Content-Language: fr; X-Iplb-Instance: 55968; Vary: Accept-Encoding; Date: Wed, 07 Feb 2024 19:22:27 GMT; Content-Type: text/plain; charset=UTF-8; Expires: Wed, 07 Feb 2024 19:23:26 GMT; X-Cdn-Pop-Ip: 51.68.15.224/27; Accept-Ranges: bytes | body: nohost
2024-02-07T19:22:27Z ERROR unknown response received: nohost
```

Configuration file (**remove your credentials!**):

```json
{
  "settings": [
    {
      "provider": "ovh",
      "domain": "domain.com",
      "host": "sub",
      "username": "username",
      "password": "password",
      "ip_version": "ipv4",
      "provider_ip": true
    }
  ]
}
```

Host OS:
ubuntu server.

**!!! Important. I noticed that the "hostname" was a bit repetitive(subdomain.domain.com.domain.com". I tried to access myself to this URL, which gave me a result of "nohost" too.**

I changed the domain in the url to be:

```https://username:password@0www.ovh.com/nic/update?hostname=subdomain.domain.com&system=dyndns```
And then it was able to work

**Comment by qdm12** [comment]
Closing this due to inactivity
